packages/react-instantsearch-core/src/core/createInstantSearchManager.js

```diff
@@ -29,10 +29,13 @@ const isIndexWidgetEqualIndex = (widget, indexId) =>
   widget.props.indexId === indexId;
 
 const sortIndexWidgetsFirst = (firstWidget, secondWidget) => {
-  if (isIndexWidget(firstWidget)) {
+  const isFirstWidgetIndex = isIndexWidget(firstWidget);
+  const isSecondWidgetIndex = isIndexWidget(secondWidget);
+
+  if (isFirstWidgetIndex && !isSecondWidgetIndex) {
```

Comment:
Why did you need to make these changes?

Reply:
Regarding that function, lots of tests failed. I searched for it and there has been a change in the implementation of Array.sort() in Node v11 ([link](https://github.com/nodejs/node/issues/24294)).
And the previous implementation of `sortIndexWidgetsFirst` didn't consider the both widgets being index. After fixing them like this, all the tests passed.